holoviews/plotting/bokeh/callbacks.py

```diff
@@ -152,39 +164,84 @@ def __init__(self, plot, streams, source, **params):
         self.streams = streams
         self.comm = self._comm_type(plot, on_msg=self.on_msg)
         self.source = source
+        self.handle_ids = defaultdict(list)
 
 
     def initialize(self):
         plots = [self.plot]
         if self.plot.subplots:
             plots += list(self.plot.subplots.values())
 
```

Comment:
I am also assuming there is a good reason to use update... i.e there may be other ``handle_ids`` on the ``Callback`` other than the ones currently returned by ``_get_handle_ids``. If that is true, it is a bit strange to see this in an ``initialize`` method which sounds like it should only happen once.